app/src/main/java/com/opsat7373/moviestars/adapters/MovieItemAdapter.kt

```diff
@@ -25,12 +29,29 @@ class MovieItemAdapter() : RecyclerView.Adapter<MovieItemAdapter.MovieItemViewHo
     }
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieItemViewHolder {
-        val itemView = LayoutInflater.from(parent.context).inflate(R.layout.movie_item, parent, false)
+        val itemView = LayoutInflater.from(parent.context).inflate(
+            R.layout.movie_item,
+            parent,
+            false
+        )
         return MovieItemViewHolder(itemView)
     }
 
     override fun onBindViewHolder(holder: MovieItemViewHolder, position: Int) {
-        holder.binding.movieName.text = moviesList[position].original_title
+        holder.binding.apply {
+            movieName.text = moviesList[position].original_title
+            movieRating.text = moviesList[position].vote_average
+            Glide
```

Comment:
Move the picture loading to a separate method.
For future: create databinding adapter that uses Glide to load a picture to an image view